tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -937,4 +937,16 @@
     <value>X64 process running on X64 machine</value>
     <comment>Message text when the process is x64 CPU architecture running on an X64 CPU machine.</comment>
   </data>
+  <data name="AliasDisabledDialogButtonText" xml:space="preserve">
+    <value>Go to Advanced app settings</value>
+    <comment>Content Dialog button text which on click opens the Advanced App settings page in Windows Settings app.</comment>
+  </data>
+  <data name="AliasDisabledDialogContent" xml:space="preserve">
+    <value>Go to Windows Settings &gt; Apps &gt; Advanced app settings &gt; App execution alias &gt; Project Ironsides &gt; Enable the toggle and try to launch PI as admin again.</value>
+    <comment>Locked="{Project Ironsides}" Content Dialog content suggesting users steps to enable app execution alias from Windows Settings app</comment>

```

Comment:
@krschau your changes locked only "Ironsides"  What is the correct locking behavior?

Reply:
I locked only "Ironsides" at @cinnamon-msft's suggestion (https://github.com/microsoft/devhome/issues/3155#issuecomment-2155251557). Happy to do either, but agree we should consolidate.

Reply:
Got you, I don't have a preference. Will update to lock only Ironsides to keep it consistent with @krschau's changes.